lib/doorkeeper/oauth/error_response.rb

```diff
@@ -12,6 +12,7 @@ def self.from_request(request, attributes = {})
           attributes.merge(
             name: error_name_for(request.error),
             exception_class: exception_class_for(request.error),
+            translate_options: request.error.try(:translate_options),
```

Comment:
I was not completely confident what classes `request.error` are expected here.  I can remove the `try` if we always expect to receive a `DoorkeeperError` here.

Reply:
Maybe there's a default that can be set? grab :translate_options or fallback to {}

Reply:
I default it [here](https://github.com/doorkeeper-gem/doorkeeper/pull/1747/files#diff-990b33ab855d22431465363199d973eb7dd1c5eb2dbfbfa0151801ada778bdf5R7) in `error.rb` because that caught all the usages of `Error` in one place.

Reply:
More thinking of if the Error somehow doesn't derive from Doorkeeper::OAuth::Error, e.g., it comes from activerecord or something